charts/zenith-proxy/README.md

```diff
@@ -53,5 +54,3 @@ Kubernetes: `^1.18.x-x`
 | settings.uri | string | `""` |  |
 | tolerations | list | `[]` | Tolerations for pod assignment. |
 
-----------------------------------------------
-Autogenerated from chart metadata using [helm-docs v1.5.0](https://github.com/norwoodj/helm-docs/releases/v1.5.0)
```

Comment:
I compiled `helm-docs` from source from `tags/v1.5.0`, ran it, and it deleted this line. I'm looking into what I did wrong..